### ISSUE TITLE
Side Navigation - bottom

### ISSUE THREAD

**Issue by umeli** [HCL-TECH-SOFTWARE/connections-ui-docs#6]
Is the submenu supposed to work in the bottom location ? I've got a menu with a submenu in the location "main", when I move it to the bottom, the submenu wont open.

**Comment by caiaga** [comment]
You are correct, the submenu is not currently intended to be used (or possible to use) in the bottom. This is not clearly stated in our documentation, so I will plan for a correction there.
Would the addition of this be desired/critical for certain use cases in your opinion?

**Comment by umeli** [comment]
From my opinion, it just needs to be clear.  Either document, that it's not possible or make it possible. It would be nice to have. As it would allow to add more elements on lower resolutions/screen sizes
